go.mod

```diff
@@ -2,11 +2,8 @@ module github.com/fergusstrange/embedded-postgres
 
 go 1.13
 
-// To avoid CVE CVE-2021-29482
-replace github.com/ulikunitz/xz => github.com/ulikunitz/xz v0.5.8
-
 require (
 	github.com/lib/pq v1.8.0
-	github.com/mholt/archiver/v3 v3.5.0
 	github.com/stretchr/testify v1.6.1
+	github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8
```

Comment:
I'm a bit worried that you're going to use an xz library from 2017 (!!).
It seems like https://github.com/ulikunitz/xz is more recommended... (it's even recommended on the README.md of the above library)

Reply:
Actually this is the library we've been using under the hood from [mholt archiver](https://github.com/mholt/archiver/blob/d35d4ce7c5b2411973fb7bd96ca1741eb011011b/xz.go#L28), and is a lighter weight version of ulikunitz/xz when we only need decompression. The README suggests to use ulikunitz/xz when compression is needed, which we can get away without.

This should help us avoid any issues on migration as like decompression algorithm won't have changed, and avoid having to track any of the CVEs which keep cropping up in mholt/archiver. Happy to assess this moving forward if this one crops up in any vulnerability scans or there are issues but it appears to have been a pretty rock solid library for that long!